### DISCUSSION TITLE
Was a happy user, not so after upgrade ...

### DISCUSSION THREAD

**Discussion by mjacobs3761** [tagyoureit/nodejs-poolController#669] [App and O/S Config Questions]
So was running older version nodejs, npm and a 6.x version of PoolController.

Upgraded to 7,7, did do on a new raspberry with nodejs v18.12.1 and npm v9.2.0.
Running npm install, I get the following error messages... anybody ever seen this or can give me a direction as to what is wrong?

root@Penty:/home/desktop/Pool_Controller/nodejs-poolController-7.6.1# npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: eslint-config-standard@16.0.3
npm ERR! Found: eslint@8.8.0
npm ERR! node_modules/eslint
npm ERR!   dev eslint@"^8.8.0" from the root project
npm ERR!   peer eslint@"^6.0.0 || ^7.0.0 || ^8.0.0" from @typescript-eslint/eslint-plugin@5.11.0
npm ERR!   node_modules/@typescript-eslint/eslint-plugin
npm ERR!     dev @typescript-eslint/eslint-plugin@"^5.11.0" from the root project
npm ERR!   10 more (@typescript-eslint/parser, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer eslint@"^7.12.1" from eslint-config-standard@16.0.3
npm ERR! node_modules/eslint-config-standard
npm ERR!   dev eslint-config-standard@"^16.0.3" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: eslint@7.32.0
npm ERR! node_modules/eslint
npm ERR!   peer eslint@"^7.12.1" from eslint-config-standard@16.0.3
npm ERR!   node_modules/eslint-config-standard
npm ERR!     dev eslint-config-standard@"^16.0.3" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /root/.npm/_logs/2022-12-28T00_42_48_640Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-12-28T00_42_48_640Z-debug-0.log

And then npm start gives me the following error message:

root@Penty:/home/desktop/Pool_Controller/nodejs-poolController-7.6.1# npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: eslint-config-standard@16.0.3
npm ERR! Found: eslint@8.8.0
npm ERR! node_modules/eslint
npm ERR!   dev eslint@"^8.8.0" from the root project
npm ERR!   peer eslint@"^6.0.0 || ^7.0.0 || ^8.0.0" from @typescript-eslint/eslint-plugin@5.11.0
npm ERR!   node_modules/@typescript-eslint/eslint-plugin
npm ERR!     dev @typescript-eslint/eslint-plugin@"^5.11.0" from the root project
npm ERR!   10 more (@typescript-eslint/parser, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer eslint@"^7.12.1" from eslint-config-standard@16.0.3
npm ERR! node_modules/eslint-config-standard
npm ERR!   dev eslint-config-standard@"^16.0.3" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: eslint@7.32.0
npm ERR! node_modules/eslint
npm ERR!   peer eslint@"^7.12.1" from eslint-config-standard@16.0.3
npm ERR!   node_modules/eslint-config-standard
npm ERR!     dev eslint-config-standard@"^16.0.3" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /root/.npm/_logs/2022-12-28T00_42_48_640Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-12-28T00_42_48_640Z-debug-0.log

**Comment by tagyoureit** [comment]
Use Node 12/14 for ver 7. Ver 8 will be out shortly and support Node 16/18.

**Comment by mjacobs3761** [comment]
Awesome, thanks, will downgrade

On Tue, Dec 27, 2022 at 7:57 PM tagyoureit ***@***.***> wrote:

> Use Node 12/14 for ver 7. Ver 8 will be out shortly and support Node 16/18.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/tagyoureit/nodejs-poolController/discussions/669#discussioncomment-4505546>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJWWKDOI4JKJO6F7FUOGNALWPOF6VANCNFSM6AAAAAATK2PFFM>
> .
> You are receiving this because you authored the thread.Message ID:
> <tagyoureit/nodejs-poolController/repo-discussions/669/comments/4505546@
> github.com>
>
-- 


Greetings,

Michel Jacobs
***@***.***
+1 469 831 3537
